### ISSUE TITLE
dynamic props are broken

### ISSUE THREAD

**Issue by TrejGun** [lodash/babel-plugin-lodash#39]
very useful plugin but broken :(

code
```js
"use strict";

export default {
	[process.env.NODE_ENV]: process.env.NODE_ENV,
	test: "test"
};
```

exits with error
```
➜  operator git:(operator) ✗ node_modules/.bin/babel test.js
TypeError: test.js: Cannot read property 'name' of null
    at PluginPass.<anonymous> (/operator/node_modules/babel-plugin-lodash/lib/index.js:152:28)
    at newFn (/operator/node_modules/babel-traverse/lib/visitors.js:262:19)
    at NodePath._call (/operator/node_modules/babel-traverse/lib/path/context.js:63:18)
    at NodePath.call (/operator/node_modules/babel-traverse/lib/path/context.js:47:17)
    at NodePath.visit (/operator/node_modules/babel-traverse/lib/path/context.js:93:12)
    at TraversalContext.visitQueue (/operator/node_modules/babel-traverse/lib/context.js:146:16)
    at TraversalContext.visitMultiple (/operator/node_modules/babel-traverse/lib/context.js:110:17)
    at TraversalContext.visit (/operator/node_modules/babel-traverse/lib/context.js:176:19)
    at Function.traverse.node (/operator/node_modules/babel-traverse/lib/index.js:131:17)
    at NodePath.visit (/operator/node_modules/babel-traverse/lib/path/context.js:103:22)
```

.babelrc
```
{
  "presets": [
    "es2015"
  ],
  "plugins": [
    "lodash"
  ]
}
```

**Comment by wouterverweirder** [comment]
Got an issue probably related to this one (error on exact same line). When using spreads in a jsx, without any additional props, it works fine:

`<PostForm {...post} />`

As soon as I add additional arguments, e.g.

`<PostForm {...post} mode="edit" />`

My build breaks:

`ERROR in ./_js/pages/Overview.jsx
Module build failed: TypeError: /Users/wouter/_js/pages/Overview.jsx: Cannot read property 'name' of null
    at PluginPass.<anonymous> (/Users/wouter/node_modules/babel-plugin-lodash/lib/index.js:152:28)`

**Comment by MadaraUchiha** [comment]
I've managed to reproduce this with only `var foo;` in my JS file. It sounds like a critical bug that affects everyone.

**Comment by MadaraUchiha** [comment]
@megawac When is a release with this fixed revision expected to be released? We're currently working with 2.1.0 but we get build warnings about not being in the latest stable version ;)

**Comment by megawac** [comment]
Couple minutes, just fixing another regression (#37)

**Comment by megawac** [comment]
Fixed in 2.2.1
